### PR TITLE
Mnemonic input for Cosmos Client

### PR DESCRIPTION
**Pull request by expertdicer**
This Pull request includes: 

- Change client key input, you now only need to input ``mnemonic`` instead of all 4 items ``private key``, ``public key``, ``account``, ``address``.
- Test cases for the ``mnemonic`` feature.

### CODE REVIEW

File: hyperspace/cosmos/src/lib.rs [hunk 1]
```diff
@@ -23,11 +23,13 @@ pub mod events;
 pub mod key_provider;
 pub mod light_client;
 pub mod provider;
-#[cfg(any(test, feature = "testing"))]
-pub mod test_provider;
+
+// #[cfg(any(test, feature = "testing"))]
+// pub mod cosmos_test_provider;
```
Comment (vmarkushin) [inline hyperspace/cosmos/src/lib.rs:28]: Not needed anymore, I guess

**Review by vmarkushin — APPROVED** [review]
Good PR, thank you!

File: hyperspace/cosmos/src/lib.rs (hunk 1 above)
Comment (expertdicer) [reply hyperspace/cosmos/src/lib.rs:28]: The ``test_provider`` makes the unit test not working. The comment thing is only for testing purpose. Let me uncomment it.

**Comment by expertdicer** [comment]
hey @blasrodri , I wonder which branch I should push my code to ?

**Comment by blasrodri** [comment]
> hey @blasrodri , I wonder which branch I should push my code to ?

Try main branch

**Pull request closed without merging**